2DW28WSQJ/note.json

```diff
@@ -0,0 +1,290 @@
+{
+  "paragraphs": [
+    {
+      "text": "%md\nThis notebook is used to perform check that Zeppelin is functioning properly.\n\u003cB\u003ePLEASE DON NOT DELETE OR MODIFY THIS NOTEBOOK \u003c/B\u003e",
+      "user": "splice",
+      "dateUpdated": "2018-10-31 01:34:01.082",
+      "config": {
+        "colWidth": 12.0,
+        "fontSize": 9.0,
+        "enabled": true,
+        "results": {},
+        "editorSetting": {
+          "language": "markdown",
+          "editOnDblClick": true,
+          "completionSupport": false
+        },
+        "editorMode": "ace/mode/markdown",
+        "editorHide": false,
```

Comment:
editorHide should be true on this paragraph

Reply:
Done